target/linux/rtl838x/files-5.4/drivers/gpio/Makefile

```diff
@@ -0,0 +1,172 @@
+# SPDX-License-Identifier: GPL-2.0
```

Comment:
Please do not overwrite this file but add a patch

Reply:
Will do.

Reply:
Done.